cypress_test/integration_tests/desktop/writer/track_changes_spec.js

```diff
@@ -8,8 +8,11 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Track Changes', function (
 	var testFileName;
 
 	beforeEach(function () {
+		cy.viewport(1400, 600);
```

Comment:
Why are the viewport and zoom changes necessary?

Reply:
To make comments fully visible